labeler/has_pull_request.go

```diff
@@ -0,0 +1,47 @@
+package labeler
+
+import (
+	"log"
+	"regexp"
+	"strconv"
+
+	"github.com/google/go-github/github"
+	"github.com/parkr/auto-reply/ctx"
+)
+
+var fixesIssueMatcher = regexp.MustCompile(`Fixes #(\d+)`)
```

Comment:
Yeah. Would you mind helping me to add some unit tests around this stuff? Just so we don't break the "Fixes" case. 'Cause I tested that one manually....